### DISCUSSION TITLE
Document @types handling

### DISCUSSION THREAD

**Discussion by rarkins** [renovatebot/renovate#21657] [Suggest an Idea]
Add a note if we implicitly grouped a types package? Perhaps linking to documentation?

**Comment by AdrienLemaire** [comment]
Is this task related to typescript DefinitelyTyped @types/ packages ?
I noticed that none of the PR requests offer release notes regarding the changes in types, and the source is always https://renovatebot.com/gh/DefinitelyTyped/DefinitelyTyped, which isn't useful.

If we look at the npm package (let's say [@types/react-dom](https://www.npmjs.com/package/@types/react-dom), There's always a `Files were exported from` link, often from DefinitelyTyped, sometimes not.
But even this doesn't give much information as to what has changed.

Do you have any tip regarding types release notes ?

**Comment by rarkins** [comment]
That URL is just the way we redirect to https://github.com/DefinitelyTyped/DefinitelyTyped

Last I checked, none of the types packages publish changelogs, although there was a discussion of allowing individual packages to publish their own Changelog file within the subdirectory.

I suppose we could add special logic just for this repo to point to the subdirectory, e.g. https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/react-dom - if you think that helps a little. But you're probably not going to find changelogs.
